server/constants/appointment-status.json

```diff
@@ -0,0 +1,5 @@
+{
+    "upcoming": "upcoming",
+    "completed": "completed",
+    "cancelled": "cancelled"
+}
```

Comment:
I double checked with the design document, but am I missing any other appointment statuses?

Reply:
Verified the appointment statuses and they include upcoming, completed and cancelled - there are none missing from your part.

Reply:
Sounds good, thank you for the verification!